frontend/packages/policy-editor/src/components/PolicyCardRules/PolicyRule/PolicyAccessPackages/PolicyAccessPackages.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { type ReactElement, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  StudioAlert,
+  StudioLabelAsParagraph,
+  StudioParagraph,
+  StudioTextfield,
+} from '@studio/components';
+import { getUpdatedRules } from '../../../../utils/PolicyRuleUtils';
+import { usePolicyEditorContext } from '../../../../contexts/PolicyEditorContext';
+import { usePolicyRuleContext } from '../../../../contexts/PolicyRuleContext';
+import classes from './PolicyAccessPackages.module.css';
+import { PolicyAccessPackageAccordion } from './PolicyAccessPackageAccordion';
+import { PolicyAccordion } from './PolicyAccordion';
+import {
+  filterAccessPackagesBySearchString,
+  groupAccessPackagesByArea,
+} from './policyAccessPackageUtils';
+import type { PolicyAccessPackageArea } from '@altinn/policy-editor';
```

Comment:
I see that we some places use absolute and some places we use relative path imports within the same files, can we please stick to one way? 😄